### ISSUE TITLE
RFE: Add camping layer

### ISSUE THREAD

**Issue by Talkless** [hholzgra/maposmatic#127]
When rendering bigger maps, you loose details like [tourism=picnic_site](https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dpicnic_site), [amenity=shelter](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dshelter), [leisure=firepit](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dfirepit), [leisure=picnic_table](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpicnic_table) that would be very useful when hiking with map & compass.

**Comment by hholzgra** [comment]
Which specific style is this about?

Anyway, I'm not really involved in maintaining any of the styles I offer, just some of the simple overhead styles were created by me.

I'm just using upstream styles, and only really modify them to adjust for database schema issues.

Most of the styles were created with rather low resolution computer screens in mind, so the detail levels are not as good as they could be for higher resolution print output. 

But as I'm not an expert in creating map styles at all I'm not really the right person to ask about this in the end. 

My MapOSMatic fork only tries to integrate as many existing open source styles as possible, but not to modify them or create new ones from scratch.

**Comment by Talkless** [comment]
Wait, so if I select "Security cameras" layer for example, and I get cameras rendered on top of map, these cameras comes from some external source? They are not rendered by maposmatic?

**Comment by hholzgra** [comment]
No, everything is rendered by the get-map.org server locally. It has all those styles installed and local copies of the OSM database and NASA SRTM based contour data.

But it is only *using* those map styles, most of them are actually maintained by other people in separate project.

(The camera overlay is one of the few exceptions that are indeed maintained within the renderer code base itself as the original "Surveillance under Surveillance" is using Javascript to render directly into a browser canvas, so I did re-recreate the camera style ...)
